frontend/src/metabase/query_builder/components/data-search/SearchResults.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { t } from "ttag";
+
+import Icon from "metabase/components/Icon";
+
+import Search from "metabase/entities/search";
+
+import { SearchResultItem } from "./SearchResultItem";
+
+const MAX_SEARCH_RESULTS = 200;
```

Comment:
@kdoh what do you think about limiting search results to some meaningful number?

Reply:
this is done already in backend btw, `limit` sets all official-like a limit